configure.ac

```diff
@@ -2221,7 +2221,7 @@ if test -z "$ssl_backends" -o "x$OPT_GNUTLS" != xno; then
 fi
 
 dnl ---
-dnl Check which crypto backend GnuTLS uses
+dnl Check which crypto backend GnuTLS uses, only nettle is supported
```

Comment:
I forgot to remove this change when rebasing, I will remove that before pushing to master as it's pointless churn.